src/pybind/mgr/dashboard/module.py

```diff
@@ -751,20 +752,110 @@ def get_perf_schema(self, **args):
                         ret[k1][k2] = sorted_dict
                 return ret
 
-            #Adding method to consolidate the data of all osd into one JSON format.
+            #Method to consolidate the data of all osd into one JSON format.
             @cherrypy.expose
             @cherrypy.tools.json_out()
             def get_counter_allosd(self, path):
                 osd_map = global_instance().get("osd_map")
-                logger.debug(osd_map)
-                result = {}
+                result = []
                 for osd in osd_map['osds']:
                     osd_id = "{0}".format(osd['osd'])
                     counter_data = global_instance().get_counter('osd', osd_id, path)
-                    result[osd['osd']] = counter_data[path]
+                    result.append(counter_data[path])
+
+                return result
+
+            #Method to find the average of consolidated data
```

Comment:
s/Method to /

Reply:
It is fixed in the latest commit. Perhaps, I should squash all the changes related to graphs in a single commit.

Reply:
@Saumay-Agrawal commits should be self-contained and be atomic. please do so.